defs.bzl

```diff
@@ -16,5 +16,5 @@
 
 Users should not load files under "/internal"
 """
-load("//internal:node.bzl", "nodejs_binary")
 load("//internal:node_install.bzl", "node_repositories")
+load("//internal:node.bzl", "nodejs_binary")
```

Comment:
is this change intentional?

Reply:
just changing the order to be more intuitive, but it has no effect on the PR, I'll remove for clarity